packages/reaction-core/server/methods/orders.js

```diff
@@ -160,35 +160,42 @@ Meteor.methods({
    *
    * @summary trigger shipmentShipped status and workflow update
    * @param {Object} order - order object
-   * @return {Object} return workflow result
+   * @param {Object} shipment - shipment object
+   * @return {Object} return results of several operations
    */
-  "orders/shipmentShipped": function (order) {
+  "orders/shipmentShipped": function (order, shipment) {
     check(order, Object);
     this.unblock();
 
-    if (order) {
-      let shipment = order.shipping[0];
+    let completedItemsResult;
+    let completedOrderResult;
 
-      // Attempt to sent email notification
-      Meteor.call("orders/sendNotification", order);
+    // Attempt to sent email notification
+    const notifyResult = Meteor.call("orders/sendNotification", order);
```

Comment:
Partially it returns `true`. And we could also refactor `orders/sendNotification` to make it return something, because I think this is important to admin to know was notification really sent or not.